x-pack/filebeat/processors/decode_cef/cef/parser.go

```diff
@@ -829,6 +825,7 @@ func (e *Event) unpack(data []byte) error {
 		recoveredErrs = append(recoveredErrs, fmt.Errorf("malformed value for %s at pos %d", extKey, p+1))
 		(p)--
 		cs = 28
+		goto _again
 
 		goto _again
```

Comment:
does this compile?

Reply:
Oh, I only accidentally commited these files. I was just testing `ragel` installed using `apt`. I am reverting this file.